modules/phase_field/src/postprocessors/FindValueOnLine.C

```diff
@@ -0,0 +1,114 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "FindValueOnLine.h"
+#include "MooseMesh.h"
+
+template<>
+InputParameters validParams<FindValueOnLine>()
+{
+  InputParameters params = validParams<GeneralPostprocessor>();
+  params.addParam<Point>("start_point", "Start point of the sample line.");
+  params.addParam<Point>("end_point", "End point of the sample line.");
+  params.addParam<Real>("target", "Target value to locate.");
+  params.addParam<unsigned int>("depth", "Number of bisections to perform.");
```

Comment:
Do you want a default value of depth?   Or maybe you could work out "depth" by inputting instead an acceptable distance tolerance for the error in the point's position?